exp/addons/controllers/clusterresourceset_controller.go

```diff
@@ -387,3 +411,29 @@ func (r *ClusterResourceSetReconciler) clusterToClusterResourceSet(o handler.Map
 	}
 	return result
 }
+
+// resourceToClusterResourceSet is mapper function that maps resources to ClusterResourceSet
+func (r *ClusterResourceSetReconciler) resourceToClusterResourceSet(o handler.MapObject) []ctrl.Request {
+	result := []ctrl.Request{}
+	acc, err := meta.Accessor(o.Object)
```

Comment:
This returns the same resource that we already have with o.Meta (a metav1.Object), so I believe this step can be skipped in favor of using `o.Meta` in place of `acc`